reference/shipping.v3.yml

```diff
@@ -142,6 +143,61 @@ paths:
                     CA: '508313'
                     AU: '817355'
                     ALL: '501000'
+
+  '/shipping/settings':
+    parameters:
+      - $ref: '#/components/parameters/Accept'
+
+    get:
+      operationId: getCheckoutSettings
```

Comment:
I think I prefer the language of shippingSettings rather than checkoutSettings

Reply:
Thanks Andrew, that's right, I forgot checkout is only one setting of shipping. I've updated all to `Shipping Settings`🍻